src/extension.ts

```diff
@@ -221,7 +221,8 @@ function startClient(context: vscode.ExtensionContext) {
 				vscode.workspace.createFileSystemWatcher("**/*.d"),
 				vscode.workspace.createFileSystemWatcher("**/dub.json"),
 				vscode.workspace.createFileSystemWatcher("**/dub.sdl"),
-				vscode.workspace.createFileSystemWatcher("**/profilegc.log")
+				vscode.workspace.createFileSystemWatcher("**/profilegc.log"),
+				vscode.workspace.createFileSystemWatcher("**/compile_commands.json"),

```

Comment:
this watcher won't do anything without it there being event handlers in serve-d for it. (e.g. requests `textDocument/didChange`, `workspace/didCreateFiles`, `workspace/didChangeWatchedFiles`, `workspace/didDeleteFiles` can be handled for this)

Reply:
Thanks, I will look into this.